staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -366,7 +366,7 @@ func (r *Reflector) RunWithContext(ctx context.Context) {
 
 var (
 	// nothing will ever be sent down this channel
-	neverExitWatch <-chan time.Time = make(chan time.Time)
+	neverExitWatch <-chan time.Time
```

Comment:
let's delete this variable entirely in favor of returning `nil` in the single place where this was used (can move the comment to that line)

also make the same fix in staging/src/k8s.io/apiserver/pkg/endpoints/handlers/watch.go

Reply:
done